resources/qml/UserProfile.qml

```diff
@@ -0,0 +1,258 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
+import QtQuick.Window 2.3
+
+import im.nheko 1.0
+
+import "./device-verification"
+
+ApplicationWindow{
+	property var profile
+
+	id: userProfileDialog
+	height: 650
+	width: 420
+	modality: Qt.WindowModal
+	Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+	palette: colors
+
+	Connections{
+		target: deviceVerificationList
+		onUpdateProfile: {
+			profile.fetchDeviceList(profile.userid)
+		}
+	}
+
+	Component {
+		id: deviceVerificationDialog
+		DeviceVerification {}
+	}
+	Component{
+		id: deviceVerificationFlow
+		DeviceVerificationFlow {}
+	}
+
+	background: Item{
```

Comment:
Why is this the background?

Reply:
I really don't remember why i did that. Anyways I removed it.